### PR TITLE
UV-10 -Update algo for cocos

### PR DESCRIPTION
**Pull request by FilipCivljak**
resolves: #10

### CODE REVIEW

**Review by SammyOina — COMMENTED** [review]
please update readme with instructions for where to place datasets once downloaded

**Review by SammyOina — CHANGES_REQUESTED** [review]
add a requirements file

**Review by SammyOina — CHANGES_REQUESTED** [review]
fix typos in readme file, also include step by step actions to take when testing with cocos

File: requirements.txt
Comment (SammyOina) [inline requirements.txt:1]: limit to minimum requirements required for file
```
numpy==2.0.0
pandas==2.2.2
scikit-learn==1.5.1
torch==2.3.1
joblib==1.4.2
matplotlib==3.9.1
```

**Review by SammyOina — CHANGES_REQUESTED** [review]

File: rul-turbofan/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+numpy==2.0.0
+pandas==2.2.2
+scikit-learn==1.5.1
+torch==2.3.1
+joblib==1.4.2
+matplotlib==3.9.1
+seaborn==0.13.2
```
Comment (SammyOina) [inline rul-turbofan/requirements.txt:7]: add an empty line at the end

**Review by SammyOina — CHANGES_REQUESTED** [review]

**Review by rodneyosodo — CHANGES_REQUESTED** [review]
Update workflows

- https://github.com/ultravioletrs/ai/blob/main/.github/dependabot.yaml
- https://github.com/ultravioletrs/ai/blob/main/.github/workflows/python.yaml

**Comment by dborovcanin** [comment]
@FilipCivljak Please rebase and resolve comments.

**Comment by dborovcanin** [comment]
@SammyOina Please review.

**Comment by drasko** [comment]
@SammyOina please review

**Comment by drasko** [comment]
@SammyOina please review

File: .github/workflows/python.yaml [hunk 2]
```diff
@@ -7,13 +7,15 @@ on:
     paths:
       - "covid19/**"
       - "fraud-detection/**"
+      - "rul-turbofan"/**"
```
Comment (SammyOina) [inline .github/workflows/python.yaml:10]:
```suggestion
      - "rul-turbofan/**"
```

File: .github/workflows/python.yaml [hunk 3]
```diff
@@ -7,13 +7,15 @@ on:
...
       - ".github/**"
   pull_request:
     branches:
       - main
     paths:
       - "covid19/**"
       - "fraud-detection/**"
+      - "rul-turbofan"/**"
```
Comment (SammyOina) [inline .github/workflows/python.yaml:18]:
```suggestion
      - "rul-turbofan/**"
```

**Review by SammyOina — CHANGES_REQUESTED** [review]

File: rul-turbofan/README.md [hunk 4]
```diff
@@ -91,3 +91,134 @@ In this plot:
...
+```
+
+Test the model with the test data:
+
+```bash
+cd ../ai/rul-turbofan
+```
+
+```bash
+unzip results.zip -d results
```
Comment (rodneyosodo) [inline rul-turbofan/README.md:224]: Close the code block

File: rul-turbofan/README.md [hunk 5]
```diff
@@ -91,3 +91,134 @@ In this plot:
...
+  go run ./cmd/cli/main.go checksum  ../ai/rul-turbofan/datasets/RUL_FD001.txt
+  ```
+
+  ```bash
+  go run ./cmd/cli/main.go checksum  ../ai/rul-turbofan/datasets/train_FD001.txt
+  ```
+
+   ```bash
+  go run ./cmd/cli/main.go checksum  ../ai/rul-turbofan/datasets/test_FD001.txt
+  ```
```
Comment (rodneyosodo) [inline rul-turbofan/README.md:171]: Add new line after the code block

**Review by rodneyosodo — CHANGES_REQUESTED** [review]

**Review by SammyOina — APPROVED** [review]

**Pull request merged**